modules/samples-operator-crd.adoc

```diff
@@ -15,14 +15,14 @@ parameters.
 
 .Procedure
 
-*  Access the Samples Operator configuration:
+*  Access the  Cluster Samples Operator configuration:
 +
 [source, terminal]
 ----
 $ oc get configs.samples.operator.openshift.io/cluster -o yaml
 ----
 +
-The Samples Operator configuration resembles the following example:
+The Cluster Samples Operator configuration resembles the following example:
 +
 [source,yaml]
```

Comment:
The source yaml need modify too.`projectName` need removed from the yaml.

apiVersion: samples.operator.openshift.io/v1
kind: Config
projectName: cluster-samples-operator